src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/PersistableTypeBuilder.cs

```diff
@@ -0,0 +1,189 @@
+        protected override PropertyBuilder DefinePropertyCore(string name, PropertyAttributes attributes, CallingConventions callingConvention, Type returnType, Type[]? returnTypeRequiredCustomModifiers, Type[]? returnTypeOptionalCustomModifiers, Type[]? parameterTypes, Type[][]? parameterTypeRequiredCustomModifiers, Type[][]? parameterTypeOptionalCustomModifiers) => throw new NotImplementedException();
+        protected override ConstructorBuilder DefineTypeInitializerCore() => throw new NotImplementedException();
+        protected override FieldBuilder DefineUninitializedDataCore(string name, int size, FieldAttributes attributes) => throw new NotImplementedException();
+        protected override bool IsCreatedCore() => throw new NotImplementedException();
+        protected override void SetCustomAttributeCore(ConstructorInfo con, byte[] binaryAttribute) => throw new NotImplementedException();
+        protected override void SetCustomAttributeCore(CustomAttributeBuilder customBuilder) => throw new NotImplementedException();
+
+        protected override void SetParentCore([DynamicallyAccessedMembers((DynamicallyAccessedMemberTypes)(-1))] Type? parent)
+        {
+            if (parent != null)
+            {
+                if (parent.IsInterface)
+                    throw new ArgumentException("SR.Argument_CannotSetParentToInterface");
+
+                _typeParent = parent;
+            }
+            else
+            {
+                if ((_attributes & TypeAttributes.Interface) != TypeAttributes.Interface)
+                {
+                    _typeParent = typeof(object);
+                }
+                else
+                {
+                    if ((_attributes & TypeAttributes.Abstract) == 0)
+                        throw new InvalidOperationException("SR.InvalidOperation_BadInterfaceNotAbstract");
+
+                    // there is no extends for interface class
+                    _typeParent = null;
+                }
+            }
+        }
+        public override string Name => _name;
+        public override bool IsDefined(Type attributeType, bool inherit) => throw new NotImplementedException();
+        public override object[] GetCustomAttributes(bool inherit) => throw new NotImplementedException();
+        public override object[] GetCustomAttributes(Type attributeType, bool inherit) => throw new NotImplementedException();
+        public override Type GetElementType() => throw new NotSupportedException();
+        public override string? AssemblyQualifiedName => throw new NotSupportedException();
+        public override string? FullName => throw new NotSupportedException();
+        public override string? Namespace { get; }
```

Comment:
```suggestion
        public override string? Namespace => _namespace;
```
Nit: Use same style for both Name and Namespace.